### DISCUSSION TITLE
Need a small tip to track down when there is a creature issue

### DISCUSSION THREAD

**Discussion by Tolkinz** [azerothcore/azerothcore-wotlk#18247] [General]
Hello people,

I've been on Darrowshire/light's hope/warmane and some other private worlds but am completely new to this server stuffs.
Thanks to very kind and detailed information from a youtube channel, I could successfully launch all 3 divisions of legacy wow(1.12.1, 2.4.3, 3.3.5) and I am enjoying in 3.3.5 world mostly for the moment.
* Oh, its azeroth core, btw.

When initiallizing the world, however, the below red alarm scratched my nerve so I decided to look into it.

![001](https://github.com/azerothcore/azerothcore-wotlk/assets/105026618/ee5fe43f-1ebe-43c1-b8e7-8960c42debac)

I didn't know how to fix this at first.
After a while digging into SQL database, I could figure that I can sort 2 columns at the same time so I found out the below data and fixed it.

![002](https://github.com/azerothcore/azerothcore-wotlk/assets/105026618/c0ad93f7-8fcc-4f68-bfeb-dc86792711d8)

So far, so good.
But one thing still bugs me cause I don't know at which location these two creatures are behaving.

My question is this ;

**Is there a way to actually pin down these two creatures in game and see where and how they are moving?**

If yes then it would be much better to put some reasonable numbers into that wandering distance.
Thanks for reading!

Kevin

**Comment by chaosua** [comment]
In game
.go creature 105426
.go creature 104915
select, kill, respawn, observe what it is doing =)

**Comment by Tolkinz** [comment]
Ahh okay, I could get close to the npcs and could verify movement was reasonably normal.
TYVM!
